Tests/test_pairwise_aligner.py

```diff
@@ -1823,30 +2107,37 @@ def test_three_letter_amino_acids_local(self):
         seq2 = ["Pro", "Pro", "Gly", "Ala", "Ala", "Cys", "Thr", "Leu"]
         aligner = Align.PairwiseAligner()
         aligner.mode = "local"
-        aligner.alphabet = ["Ala", "Arg", "Asn", "Asp", "Cys",
-                            "Gln", "Glu", "Gly", "His", "Ile",
-                            "Leu", "Lys", "Met", "Phe", "Pro",
-                            "Ser", "Thr", "Trp", "Tyr", "Val"]
+        aligner.alphabet = (
+            ["Ala", "Arg", "Asn", "Asp", "Cys"]
+            + ["Gln", "Glu", "Gly", "His", "Ile"]
+            + ["Leu", "Lys", "Met", "Phe", "Pro"]
+            + ["Ser", "Thr", "Trp", "Tyr", "Val"]
+        )
```

Comment:
Rather than using list + list + ... + list just use one multi-line list more like the original but with black style square brackets:

```python
aligner.alphabet = [
    "Ala", "Arg", "Asn", "Asp", "Cys",
    "Gln", "Glu", "Gly", "His", "Ile",
    "Leu", "Lys", "Met", "Phe", "Pro",
    "Ser", "Thr", "Trp", "Tyr", "Val",
]
```

If the 88 line length lets you,

```python
aligner.alphabet = [
    "Ala", "Arg", "Asn", "Asp", "Cys", "Gln", "Glu", "Gly", "His", "Ile",
    "Leu", "Lys", "Met", "Phe", "Pro", "Ser", "Thr", "Trp", "Tyr", "Val",
]
```

Reply:
No, black won't let you do that. Changing it to two lines didn't work either. It changed it to this:

```python
        aligner.alphabet = [
            "Ala",
            "Arg",
            "Asn",
            "Asp",
            "Cys",
            "Gln",
            "Glu",
            "Gly",
            "His",
            "Ile",
        ] + ["Leu", "Lys", "Met", "Phe", "Pro", "Ser", "Thr", "Trp", "Tyr", "Val"]
```

And it didn't even exceeded 88 characters.

Looks like undesirable behavior to me.

Reply:
Let's see what they the black devs say about this behavior: psf/black/issues/1241

Reply:
I was unclear, sorry. Black will insist on one line per list entry - so almost anything we do here will be black-disabled.

My suggestion is intended as a short black-like compromise which avoids the need less list addition:

```python
# fmt: off
aligner.alphabet = [
    "Ala", "Arg", "Asn", "Asp", "Cys", "Gln", "Glu", "Gly", "His", "Ile",
    "Leu", "Lys", "Met", "Phe", "Pro", "Ser", "Thr", "Trp", "Tyr", "Val",
]
# fmt: on

Reply:
Okay, I will do that.